player/command.c

```diff
@@ -2823,28 +2823,50 @@ static int mp_property_focused(void *ctx, struct m_property *prop,
     return m_property_bool_ro(action, arg, focused);
 }
 
+static int get_display_names_subkey(struct vo *vo, int action,
+                                    void *arg, enum mp_voctrl voctrl)
+{
+    switch (action) {
+        case M_PROPERTY_GET_TYPE:
+            *(struct m_option *)arg = (struct m_option){.type = CONF_TYPE_STRING_LIST};
+            return M_PROPERTY_OK;
+        case M_PROPERTY_GET: {
+            char** display_names;
+            if (vo_control(vo, voctrl, &display_names) < 1)
+                return M_PROPERTY_UNAVAILABLE;
+
+            *(char ***)arg = display_names;
+            return M_PROPERTY_OK;
+        }
+    }
+    return M_PROPERTY_NOT_IMPLEMENTED;
+}
+
 static int mp_property_display_names(void *ctx, struct m_property *prop,
                                      int action, void *arg)
 {
-    MPContext *mpctx = ctx;
-    struct vo *vo = mpctx->video_out;
+    MPContext *const mpctx = ctx;
+    struct vo *const vo = mpctx->video_out;
     if (!vo)
         return M_PROPERTY_UNAVAILABLE;
 
-    switch (action) {
-    case M_PROPERTY_GET_TYPE:
-        *(struct m_option *)arg = (struct m_option){.type = CONF_TYPE_STRING_LIST};
-        return M_PROPERTY_OK;
-    case M_PROPERTY_GET: {
-        char** display_names;
-        if (vo_control(vo, VOCTRL_GET_DISPLAY_NAMES, &display_names) < 1)
-            return M_PROPERTY_UNAVAILABLE;
-
-        *(char ***)arg = display_names;
-        return M_PROPERTY_OK;
-    }
+    if (action == M_PROPERTY_KEY_ACTION) {
+        struct m_property_action_arg *const ka = arg;
+        if (!strcmp(ka->key, "friendly")) {
+            return get_display_names_subkey(
+                vo,
+                ka->action,
+                ka->arg,
+                VOCTRL_GET_DISPLAY_NAMES_FRIENDLY);
+        } else {
+            return M_PROPERTY_UNKNOWN;
+        }
```

Comment:
```suggestion
        if (strcmp(ka->key, "friendly"))
            return M_PROPERTY_UNKNOWN;

        return get_display_names_subkey(vo, ka->action, ka->arg,
                                        VOCTRL_GET_DISPLAY_NAMES_FRIENDLY);
```

early exit and we pack arguments in the same line with continuation after (